README.md

```diff
@@ -97,7 +97,7 @@ License information of 3rd party components is available in the
 ## Trademarks
 
 - polypoly® is a registered trademark of polypoly Enterprise GmbH
-- polyPod is a trademark in registration of pc polypoly coop SCE mbH
```

Comment:
So previously the `polypod` was a trademark in registration and now it is registered?